lib/faker/gov/it.ex

```diff
@@ -0,0 +1,161 @@
+defmodule Faker.Gov.It do
+  @moduledoc """
+  Generating Italian fiscal codes
+  """
+
+  require Integer
+
+  alias Faker.Util
+
+  # @doc """
+  # Returns a random Italian fiscal code
+
+  # ## Examples
+
+  #     iex> Faker.Gov.It.fiscal_id
+  #     "ZRSNND48A23E151U"
+  #     iex> Faker.Gov.It.fiscal_id
+  #     "CCPWLC58L04I289K"
+  #     iex> Faker.Gov.It.fiscal_id
+  #     "TTKRQQ77P43I477X"
+  # """
+  @spec fiscal_id() :: binary()
+  def fiscal_id do
+    surname()
+    |> Kernel.<>(name())
+    |> Kernel.<>(birth_year())
+    |> Kernel.<>(birth_month())
+    |> Kernel.<>(birth_day())
+    |> Kernel.<>(town_code_letter())
+    |> Kernel.<>(town_code_numbers())
+    |> (&Kernel.<>(&1, cin(&1))).()
+  end
+
+  defp surname do
+    Enum.reduce(1..3, "", fn _, acc -> acc <> Util.upper_letter() end)
```

Comment:
How about:
```suggestion
    Util.join(3, &Util.upper_letter/0)
```
?

Reply:
I didn't notice that function, thanks!

Reply:
There was the problem with `Faker.Util.join/3` and its spec... I've tried to fix it